tests/test_setup_config_extended.py

```
@@ -297,6 +300,64 @@ def test_setup_downloads_both_selected(mocker, capsys):
     assert result_config["SAVE_FIRMWARE"] is True
 
 
+@pytest.mark.configuration
+@pytest.mark.unit
+def test_setup_downloads_apk_snapshot_enabled(mocker, capsys):
```

Ruff (0.15.20)

[warning] Unused function argument: `capsys`

(ARG001)